udata/harvest/tests/dcat/geonetwork.xml

```diff
@@ -41,10 +41,10 @@
           /
           2005-03-30T00:00:00+02:00</dct:temporal>
       <dct:issued xmlns:dct="http://purl.org/dc/terms/">2004-11-03</dct:issued>
-      <dct:updated xmlns:dct="http://purl.org/dc/terms/">2021-05-05</dct:updated>
+      <dct:modified xmlns:dct="http://purl.org/dc/terms/">2021-05-05</dct:modified>
```

Comment:
Should we modify that test file? I got it from a "real" geonetwork dump. Also, shouldn't we support `dct:updated` too ?

Idea: create a new test file from the real stream of OREME.

Reply:
Yep to create a new test file from stream OREME. They also use `dct:updated` instead of `dct:modified` though.

Should we support terms that are not documented in wc3: https://w3c.github.io/dxwg/dcat/#Property:resource_update_date?

Reply:
An extract from the OREME catalog to date was added as a text example.